llvm/docs/RISCVUsage.rst

```diff
@@ -271,7 +271,7 @@ The primary goal of experimental support is to assist in the process of ratifica
 ``experimental-zalasr``
   LLVM implements the `0.0.5 draft specification <https://github.com/mehnadnerd/riscv-zalasr>`__.
 
-``experimental-zfbfmin``, ``experimental-zvfbfmin``, ``experimental-zvfbfwma``
+``zfbfmin``, ``zvfbfmin``, ``zvfbfwma``
```

Comment:
We should not just remove the `experimental` prefix, these extensions still belong to `Experimental Extensions` section.

Reply:
Fixed, move these extensions to the supported list.